frontend/src/components/ThemedButtons/ThemedButton.css

```diff
@@ -5,11 +5,16 @@
 	position: relative;
 	border: none;
 	border-radius: 0.625rem;
-	padding: 0.5rem;
+	padding: 0.45rem;
```

Comment:
Can we decide on a strategy for spacings please? Previously we've used rems based on 16px base font size, so if we are going to change that, let's change it everywhere.

I suggest keeping this as 0.5rem in the meantime.